### ISSUE TITLE
Issue: ITransaction & SetTransactionName have broken links

### ISSUE THREAD

**Issue by brnhensley** [newrelic/docs-website#5569]
## How can we make our docs better?

The link to `AddCustomParameter` in these paragraphs are broken because `AddCustomParameter` is deprecated: 

> This method requires .NET agent version and .NET agent API version 8.24.244.0 or higher. It replaced the deprecated AddCustomParameter.
https://docs.newrelic.com/docs/apm/agents/net-agent/net-agent-api/itransaction/#addcustomattribute

> Unique values like URLs, page titles, hex values, session IDs, and uniquely identifiable values should not be used in naming your transactions. Instead, add that data to the transaction as a custom parameter with the AddCustomParameter() call.
https://docs.newrelic.com/docs/apm/agents/net-agent/net-agent-api/settransactionname-net-agent-api/#description

Since I don't know the best way to handle the deprecated link I made an issue instead of fixing it. We don't maintain docs for older versions of the agents/apis do we?

### Doc information (don't delete this section)

* https://docs.newrelic.com/docs/apm/agents/net-agent/net-agent-api/itransaction
* https://docs.newrelic.com/docs/apm/agents/net-agent/net-agent-api/settransactionname-net-agent-api
* **OS**: Unknown
* **Browser**: Unknown
* **Device**: Unknown

**Comment by brnhensley** [comment]
I personally would prefer that we have `AddCustomParameter()` the docs still with a warning explaining when it was deprecated (before 8.24.244.0) and what to use now instead. 

This method doesn't exist in the most recent agent, but there are still customers who don't or can't use the most recent agent. I found these broken links because I was working on a ticket yesterday for a customer who was trying to use this method and I had a hard time helping him because it was removed from the docs.
